content/docs/hello-world.md

```diff
@@ -1,50 +1,50 @@
 ---
 id: hello-world
-title: Hello World
+title: नमस्ते दुनिया
 permalink: docs/hello-world.html
 prev: cdn-links.html
 next: introducing-jsx.html
 ---
 
-The smallest React example looks like this:
+React का सबसे छोटा उदाहरण इस प्रकार है:
 
 ```js
 ReactDOM.render(
-  <h1>Hello, world!</h1>,
+  <h1>नमस्ते, दुनिया!</h1>,
   document.getElementById('root')
 );
 ```
 
-It displays a heading saying "Hello, world!" on the page.
+यह पृष्ठ पर "नमस्ते, दुनिया!" कहते हुए शीर्षक को दिखाए गा।
```

Comment:
should be पृष्ठ ke शीर्षक  par "नमस्ते, दुनिया! dikhega or something like that

Reply:
the output text should be written as such,i.e. "Hello World!",if i,m not wrong.